src/TestStack.White.UITests/UIA/AutomationElementXTest.cs

```diff
@@ -1,28 +1,22 @@
-using System.Collections.Generic;
+using NUnit.Framework;
 using TestStack.White.UIA;
 using TestStack.White.UIItems;
-using Xunit;
 
 namespace TestStack.White.UITests.UIA
 {
-    public class AutomationElementXTest : WhiteTestBase
+    [TestFixture(WindowsFramework.WinForms)]
+    [TestFixture(WindowsFramework.Wpf)]
+    public class AutomationElementXTest : WhiteUITestBase
```

Comment:
The TestFixtures should end in "Tests" and the Tests usually should end in Test().